### PR TITLE
Port the Explorer from Create React App to Next 13 (w/ App Router)

### PR DESCRIPTION
**Pull request by steveluscher**
Apologies in advance for this un-reviewable PR. We're porting Explorer from Create React App to the brand-new-released-yesterday Next 13 with a modern App Router.

In addition to the port, this iteration features:

* Unique page titles and description metadata
* Dropdowns that work
* Repaired NFToken collection pages
* Newest version of Sentry
* Tests now run with Jest
* Update to Node 18
* Migrate to `pnpm`
* Fix an issue where NFT images would transiently fail to load

## Lighthouse scores

Without actually fixing many of the underlying problems, the old site on the new infrastructure pulls better Lighthouse scores.

||Before|After|
|---|---|---|
|Homepage|image|image|
|USDC [Token page](https://explorer.solana.com/address/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v)|image|image|
|Transaction detail page|image|image|

### CODE REVIEW

**Comment by ngundotra** [comment]
Ship it. Nothing stands out as being immediately wrong

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "typescript.tsdk": "node_modules/.pnpm/typescript@5.0.4/node_modules/typescript/lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true
```
Comment (ngundotra) [inline .vscode/settings.json:3]: Did you mean to add this file?

**Comment by ngundotra** [comment]
Just checking, I see that you've renamed a bunch of files to match their paths, but other than that why are there so many diffs in +/- files (I'd expect mostly moving files)

File: .vscode/settings.json (hunk 1 above)
Comment (steveluscher) [reply .vscode/settings.json:3]: Yeah. Next 13 insists on this somehow.

Comment (ngundotra) [reply .vscode/settings.json:3]: LOL

**Review by ngundotra — APPROVED** [review]
LGTM

**Pull request merged**